### DISCUSSION TITLE
JustSign

### DISCUSSION THREAD

**Discussion by sharethewisdom** [roelderickx/connective-plugin-linux#22] [General]
> Vandaag bevindt JustSign zich nog steeds in een pilootfase en zou het de komende maanden uitgerold worden. Momenteel is het systeem nog niet in gebruik in het kader van het digitale dossier, een project van Justitie om gerechtelijke dossiers volledig digitaal te laten verlopen.

https://www.vrt.be/vrtnws/nl/2024/05/20/bpost-overheidsdienst-justitie-fod-van-quickenborne-5-miljoen/

**Comment by roelderickx** [comment]
> Een toepassing om digitaal juridische documenten te ondertekenen die miljoenen euro’s heeft gekost, zal uiteindelijk niet worden gebruikt. Dat blijkt uit antwoorden van het kabinet van minister van Justitie Paul Van Tigchelt (Open VLD).

https://www.vrt.be/vrtnws/nl/2025/01/09/digitale-handtekening-justsign/

**Reply by sharethewisdom** [reply]
> Al in december 2023 wezen leidinggevenden binnen de overheidsdienst Justitie op de mogelijkheid om de toepassing van de FOD Bosa te gebruiken.

:angry:
